ldm/invoke/CLI.py

```diff
@@ -521,121 +558,205 @@ def set_default_output_dir(opt:Args, completer:Completer):
     completer.set_default_dir(opt.outdir)
 
 
-def add_weights_to_config(model_path:str, gen, opt, completer):
-    print(f'>> Model import in process. Please enter the values needed to configure this model:')
-    print()
+def import_model(model_path:str, gen, opt, completer):
+    '''
+    model_path can be (1) a URL to a .ckpt file; (2) a local .ckpt file path; or
+    (3) a huggingface repository id
+    '''
+    model_name = None
+    
+    if model_path.startswith(('http:','https:','ftp:')) or os.path.exists(model_path):
+        model_name = import_ckpt_model(model_path, gen, opt, completer)
+    elif re.match('^[\w-]+/[\w-]+$',model_path):
```

Comment:
<img width="383" alt="Screenshot 2023-01-06 at 11 06 10" src="https://user-images.githubusercontent.com/70191398/210921796-fde59a10-7151-4068-b46f-cc0b9cb9c3af.png">

Repo names with `.`s in them fail to pass the regex check